pkg/security/podsecuritypolicy/factory.go

```diff
@@ -111,6 +117,20 @@ func createUserStrategy(opts *policy.RunAsUserStrategyOptions) (user.RunAsUserSt
 	}
 }
 
+// createGroupStrategy creates a new group strategy.
+func createGroupStrategy(opts *policy.RunAsGroupStrategyOptions) (group.GroupStrategy, error) {
+	switch opts.Rule {
+	case policy.RunAsGroupStrategyMustRunAs:
+		return group.NewMustRunAs(opts.Ranges)
+	case policy.RunAsGroupStrategyMustRunAsNonRoot:
+		return group.NewRunAsNonRoot(opts)
+	case policy.RunAsGroupStrategyRunAsAny:
+		return group.NewRunAsAny()
+	default:
+		return nil, fmt.Errorf("Unrecognized RunAsGroup strategy type %s", opts.Rule)
```

Comment:
doesn't this mean a PSP has to specify a runasgroup strategy? we cannot require this field, since it's a new field. what is the default going to be?